src/Kontent.Ai.ModelGenerator/UsedSdkInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Reflection;
+
+namespace Kontent.Ai.ModelGenerator;
+
+public class UsedSdkInfo
+{
+    public string Name { get; }
+    public string Version { get; }
+
+    public UsedSdkInfo(Type type, string name)
+    {
+        Version = type == null ? throw new ArgumentNullException(nameof(type)) : Assembly.GetAssembly(type).GetName().Version.ToString(3);
```

Comment:
What does ToString(3) do?

Reply:
It does print the semantic version (MAJOR.MINOR.PATCH) of the assembly.